app/Models/TagRelation.php

```diff
@@ -39,7 +39,7 @@ class TagRelation extends Model
 
     protected $guarded = [];
 
-    public function tag(){
+    public function tag() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->